### PR TITLE
Add an x/wasm placeholder 

### PR DESCRIPTION
**Pull request by davepuchyr**
The upstream version of wasm is not ready for prime time with cosmos-sdk v0.44.x.  We can make the future upstream merge easier by adding the minimal scaffolding for the module now.

Closes https://github.com/iov-one/starnamed/issues/151.

### CODE REVIEW

**Comment by davepuchyr** [comment]
@LeCodeurDuDimanche , the tests all pass for me on my machine.  Any idea why https://app.travis-ci.com/github/iov-one/starnamed/jobs/551041870#L680 fails?

File: x/configuration/genesis.go [hunk 1]
```diff
@@ -46,9 +46,9 @@ func DefaultGenesisState() types.GenesisState {
 		CertificateSizeMax:     10000,
 		CertificateCountMax:    3,
 		MetadataSizeMax:        86400,
-		EscrowCommission:       sdk.ZeroDec(),                                 // zero commission at first
-		EscrowBroker:           "star1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqjewks3", // so the broker is the burn address
-		EscrowMaxPeriod:        7890000 * 1e9,                                 // 3 months
+		EscrowCommission:       sdk.NewDecFromInt(sdk.NewInt(1)).QuoInt(sdk.NewInt(100)), // 1%
+		EscrowBroker:           "star1d3lhm5vtta78cm7c7ytzqh7z5pcgktmautntqv",            // msig1
```
[the remark this answers is not in the document]
Comment (davepuchyr) [reply x/configuration/genesis.go:50]: The update config test fails if the commission is zero.

**Review by davepuchyr — COMMENTED** [review]
@LeCodeurDuDimanche Why did you change this?  The values that you used are confusing to people looking at the code for the first time.  Escrows should charge a commission and the broker should definitely not be the burner address.  (I'm referring to https://github.com/iov-one/starnamed/pull/152/commits/13a5439ac9568037ea9513f931f0a792e98267d2 since this comment didn't appear where I thought it would.)

**Comment by LeCodeurDuDimanche** [comment]
@davepuchyr The idea was to remove data that was not suitable for production (i.e. test account being the escrow broker), but I will better put a real address and a 1% commission like before

**Review by LeCodeurDuDimanche — APPROVED** [review]

**Pull request merged**